src/textOnlyPackages/src/microsoft.netcore.platforms/1.0.1/dotnet_library_license.txt

```diff
@@ -0,0 +1,128 @@
+
```

Comment:
We removed these old dotnet library licenses and replaced them with MIT licenses.  See https://github.com/dotnet/source-build-reference-packages/pull/53

Reply:
Updated these.